homeassistant/components/unifiprotect/__init__.py

```diff
@@ -83,6 +90,28 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     hass.data.setdefault(DOMAIN, {})[entry.entry_id] = data_service
     hass.config_entries.async_setup_platforms(entry, PLATFORMS)
 
+    services = [
```

Comment:
It would be nicer for readers to extract the service registration and handling to a class and put that in the services.py module so that all the service code including service schemas and constants is in one place.